File: Bridge Pattern/PHP/Amodel.php
<?php

require_once "Model.php";
require_once "Type.php";

class Amodel extends Model 
{
    private Type $type;

    public function __construct(Type $type)
    {
        $this->type = $type;
    }

    public function getModel() 
    {
        return "This is an A model of type " . $this->type->getType();
    }
}

?>

File: Bridge Pattern/CS/Amodel.cs
namespace BridgePattern
{
    public class Amodel : Model
    {
        public Amodel(IType type)
        {
            this.Type = type;
        }
        
        public override string ClassInfo()
        {
            return "A model of type " + this.Type.GetInfo();
        }
    }
}